src/renderer/components/SidePanel/artboardEditable.vue

```diff
@@ -172,7 +172,8 @@ export default {
         title: artboard.title,
         id: artboard.id,
         width: artboard.width,
-        height: artboard.height
+        height: artboard.height,
+        isVisible: artboard.isVisible
```

Comment:
No changes needed.

FYI that I'm aware this is a super repetitively named function, and the `rightClickMenu()` function should ideally not need to be passed each property individually.